### DISCUSSION TITLE
When will we find the PrimeNG 18 version

### DISCUSSION THREAD

**Discussion by nazrulcsebd** [primefaces/community#2560] [PrimeNG]
Please inform us when the 18th version of PRIMENG is released. Angular 18 already released

**Comment by qwadrox** [comment]
Hi. @nazrulcsebd 

I use PrimeNg 17 with Angular 18. Recently, I migrated a large PrimeNg/Angular 12 project to 18 without any problems. (Any problem with the PrimeNg components, not the project itself 🐙 ) 

[PrimeNG v17.18.0 has been released to ensure compatibility with Angular v18 ](https://www.primefaces.org/blog/primeng-for-angular-18/)
The versioning is not matching exactly but it is working. 

Did you experience something incompatibility?
